conf/evolutions/default/94.sql

```diff
@@ -0,0 +1,10 @@
+# --- !Ups
+ALTER TABLE virtual_challenges DROP CONSTRAINT IF EXISTS CON_VIRTUAL_CHALLENGES_USER_ID_NAME;
+DROP INDEX IF EXISTS idx_challenges_parent_id_name;
```

Comment:
We had discussed removing the index but that's not necessary, remove this line

Reply:
Is there a drawback to dropping that index? It's a solution to an error when only dropping the constraint: ERROR: duplicate key value violates unique constraint "idx_challenges_parent_id_name"
  Detail: Key (parent_id, lower(name::text))=(1, tags fix challenge) already exists.